### ISSUE TITLE
Suggestion : Geographical Area and Language categories

### ISSUE THREAD

**Issue by armandokaos** [graphprotocol/mission-control-curator#273]
It could be relevant to be able to find or to support projects by their geographical area and/or by language. My first reflex using Everest was trying to discover projects from places which I'm attached with (country where I'm living, the town where I come from). Decentralization leads to local empowerment and to local cooperations, Everest could be used to find jobs in your geographical area, to organize network meeting in your native language, to build local alliances. 
Finding projects by languages could make sense for people who don't manage english properly or for people with specific interests. 
 
I want to credit Web3Slimchance for his help defining and clearing this suggestion.

**Comment by yanivtal** [comment]
Yes it would be great to have locations for projects that are geographically bound.

**Comment by armandokaos** [comment]
Thank you for all your kind replies.
